Tools/Frame_params/intel-aero-rtf.param

```diff
@@ -0,0 +1,38 @@
+AHRS_ORIENTATION,0
+ARMING_CHECK,118
+ATC_ANG_PIT_P,3.500000000000000000
+ATC_ANG_RLL_P,3.500000000000000000
+ATC_ANG_YAW_P,3.500000000000000000
+ATC_RAT_PIT_D,0.001290548942051828
```

Comment:
@lucasdemarchi, your specialty is bad jokes, right? That might explain it :-P haha